ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.11__amd64-stable.yaml

```diff
@@ -49,11 +49,10 @@ tests:
     cluster_profile: aws-qe
     env:
       BASE_DOMAIN: qe.devcluster.openshift.com
-      E2E_RUN_TAGS: '@aws-ipi and @network-openshiftsdn and not @fips'
-      TAG_VERSION: '@4.11'
+      TEST_SCENRAIOS: Security_and_Compliance
     test:
-    - chain: openshift-e2e-test-qe
-    workflow: cucushift-installer-rehearse-aws-ipi
+    - ref: openshift-extended-test
+    workflow: cucushift-installer-rehearse-aws-ipi-ovn-fips
```

Comment:
currently, the qe-app-registry not available in the profile. You need to enable it first.

Reply:
I have enabled can you review it again, thanks

Reply:
Sorry, the file I pointing to is wrong, should be https://github.com/openshift/release/blob/master/ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/provision/cucushift-installer-rehearse-aws-ipi-ovn-fips-provision-chain.yaml#L7